op-devstack/sysgo/l1_nodes.go

```diff
@@ -54,7 +73,16 @@ func (n *L1CLNode) hydrate(system stack.ExtensibleSystem) {
 	l1Net.(stack.ExtensibleL1Network).AddL1CLNode(frontend)
 }
 
+const GethExecPathEnvVar = "SYSGO_GETH_EXEC_PATH"
+
```

Comment:
Can you add this env var (and the one proposed below) to the devstack readme, next to the other env vars, here https://github.com/ethereum-optimism/optimism/tree/develop/op-devstack#environment-variables ?